onnxruntime/core/providers/cuda/cuda_execution_provider.cc

```diff
@@ -2523,20 +2523,12 @@ void CUDAExecutionProvider::RegisterAllocator(AllocatorManager& allocator_manage
 }
 
 void CUDAExecutionProvider::RegisterStreamHandlers(IStreamCommandHandleRegistry& stream_handle_registry) const {
-  if (use_ep_level_unified_stream_)
-    RegisterCudaStreamHandles(stream_handle_registry,
-                              OrtDevice::GPU,
-                              stream_,
-                              use_ep_level_unified_stream_,
-                              GetPerThreadContext().CudnnHandle(),
-                              GetPerThreadContext().CublasHandle());
-  else
-    RegisterCudaStreamHandles(stream_handle_registry,
-                              OrtDevice::GPU,
-                              stream_,
-                              use_ep_level_unified_stream_,
-                              GetPerThreadContext().CudnnHandle(),
-                              GetPerThreadContext().CublasHandle());
+  RegisterCudaStreamHandles(stream_handle_registry,
```

Comment:
>RegisterCudaStreamHandles [](http://example.com/codeflow?start=2&length=25)

nit: pls format the code properly, especially be with {} following if-else.

Reply:
used "ctrl-k, ctrl-f" and seems no changes on these lines